contracts/DummyTestContracts/Flipper.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
```

Comment:
Maybe put that in the Utils folders? Or open a dedicated test folder for test contracts..

Reply:
> Or open a dedicated test folder for test contracts..

I did. It's called DummyTestContracts :) Do you want it one level higher?

Reply:
Sorry, for some reason I thought it was the contract name haha